migration-guide/business-central.adoc

```diff
@@ -0,0 +1,129 @@
+include::../partials/attributes.adoc[]
+= Migrating from Business Central to IBM {DEV_TOOLS}
+
+== Overview
+
+Business Central was a web-based platform for authoring business automation assets such as data objects, rules, processes, cases, and planning entities. It included capabilities for monitoring, administration, and management of business automation assets.  With the release of IBM {DEV_TOOLS} as well as IBM {CANVAS}, which unlike Business Central was designed from the ground up to be cloud-native, most of these asset management capabilities have been replaced or in some cases take a different approach altogether.  This section covers migration of business assets created in Business Central so that they are able to be maintained by developers using either {DEV_TOOLS} and/or {CANVAS}.
+
+== Prerequisites
+
+The following tools are required in order to migrate from Business Central to {DEV_TOOLS}.  It is assumed that you have a current version of Business Central running in order to be able to export certain types of assets to their supported form in {PRODUCT_SHORT}:
+
+include::../partials/prerequisites.adoc[]
+
+== Business Automation Assets in Businss Central
+
+The following types of business automation assets were available within Business Central, with particular attention paid to those assets which are deprecated in v9 as well as an indication of how to manage those assets in abscense of Business Central:
+
+.Business Central Asset Types
+[cols="2,1,4,3,3"]
+|===
+| Asset Name | Asset Type | Description | v9 Supported Tools | Additional Info.
```

Comment:
v9 Supported Tools -> Version 9.0 Supported Tools 
OR
Supported Tools in Version 9.0

Additional Info. -> Additional Information

Reply:
Version 9.0 Supported Tools